sources/Outputs/include/Dyd.h

```diff
@@ -266,14 +264,17 @@ class Dyd {
   static const std::unordered_map<algo::GeneratorDefinition::ModelType, std::string>
       correspondence_lib_;  ///< Correspondance between generator model type and library name in dyd file
   static const std::unordered_map<algo::GeneratorDefinition::ModelType, std::string>
-      correspondence_macro_connector_;                           ///< Correspondence between generator model type and macro connector name in dyd file
+      correspondence_macro_connector_;  ///< Correspondence between generator model type and macro connector name in dyd file
+  static const std::unordered_map<algo::HVDCDefinition::HVDCModel, std::string>
+      hvdcModelsNames_;                                          ///< Correspondence between HVDC model and their librayr name in dyd file
   static const std::string macroConnectorLoadName_;              ///< name of the macro connector for loads
   static const std::string macroConnectorGenName_;               ///< name for the macro connector for generators
   static const std::string macroConnectorGenSignalNName_;        ///< Name for the macro connector for SignalN
   static const std::string macroStaticRefSignalNGeneratorName_;  ///< Name for the static ref macro for generators using signalN model
   static const std::string macroStaticRefLoadName_;              ///< Name for the static ref macro for loads
   static const std::string networkModelName_;                    ///< name of the model corresponding to network
   static const std::string signalNModelName_;                    ///< Name of the SignaN model
```

Comment:
SignalN (not your dev but we can correct it)

Reply:
OK